airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/user_teams.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "user_id": {
+      "type": ["null", "string"]
+    },
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
Similar to above, ids are UUIDs:

```
{
    "data": [
        {
            "id": "0bc4eac3-fc94-41ba-b28f-873c7065c814",
            "name": "TechLT"
        },
        ...
    ],
    "took": 0.004,
    "requestId": "b233e414-7b8b-4065-9dbf-915cbd1cb69c"
}
```